pydrad/configure/configure.py

```diff
@@ -126,23 +127,36 @@ def setup_initial_conditions(self, root_dir, execute=True):
             with open(os.path.join(root_dir, filename), 'w') as f:
                 f.write(filestring)
         # NOTE: make sure we have needed permissions to run compile script
-        run_shell_command(
-            ['chmod', 'u+x', 'build_initial_conditions.bat'],
-            os.path.join(root_dir, 'Initial_Conditions/build_scripts'),
-            shell=False
-        )
-        run_shell_command(
-            ['./build_initial_conditions.bat'],
-            os.path.join(root_dir, 'Initial_Conditions/build_scripts')
-        )
+		# Only do this on Unix-based systems
+        if platform.system() != 'Windows':
```

Comment:
Since this is used multiple times, I would actually consider making it a separate property on `Configure`,

```python
@property
def on_windows(self):
    return platform.system().lower() == 'windows'
```